### PR TITLE
Formatting edits on Overview section

### PR DESCRIPTION
**Pull request by antoinebisch**
Just made minor changes for ease of reading

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,13 @@ This is an explainer for a new way that browsers could enable interest-based adv
...
+The choice of what ads to show on a web page may typically be based on three broad categories of information:  
+1. First-party and contextual information (e.g., "put this ad on web pages about motorcycles")  
+2. General information about the interests of the person who is going to see the ad (e.g., “show this ad to Classical Music Lovers”)  
+3. Specific previous actions the person has taken (e.g., "offer a discount on some shoes that you left in a shopping cart")  
+
+**This document addresses category 2, ads targeting based on someone's general interests.**
```
Comment (jkarlin) [inline README.md:11]: Can you remove the bolding here? It seems unnecessary, especially given that it starts a new paragraph.

What happened to the link to turtledove at the end?

Comment (antoinebisch) [reply README.md:11]: I'll remove the bolding. 

I haven't modified the link to Trutledove, does it look broken at your end? Should this be replaced by a link to Fledge instead?

**Review by jkarlin — APPROVED** [review]

**Comment by jkarlin** [comment]
lgtm

**Comment by jkarlin** [comment]
Argh, I thought it merged.

**Pull request merged**